specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json

```diff
@@ -2582,6 +2582,249 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataFactory/factories/{factoryName}/triggers/{triggerName}/rerunTriggers/{rerunTriggerName}": {
```

Comment:
The rerun trigger addresses the requirement from customers who want to re-run existing trigger past trigger runs, in case if old input data is corrupted and etc. For example, customer created a trigger starting from 6/1/2018-12/31/2018. If the trigger started and ran for a week, then customer found input data is partially missing, and wanted to reprocess this week of data, he would use this function and create a re-run trigger for 6/12018-6/7/2018.